server/routes/testutils/appSetup.ts

```diff
@@ -58,5 +38,5 @@ function appSetup(route: Router, production: boolean): Express {
 
 export default function appWithAllRoutes({ production = false }: { production?: boolean }): Express {
   auth.default.authenticationMiddleware = () => (req, res, next) => next()
-  return appSetup(allRoutes(standardRouter(new MockUserService())), production)
+  return appSetup(allRoutes(standardRouter(new MockUserService()), new MockCommunityApiService()), production)
```

Comment:
So we're going for dependency injection and explicit mocks created by subclasses? In the hmpps-risk-assessment-ui codebase [they instead just use Jest mocking](https://github.com/ministryofjustice/hmpps-risk-assessment-ui/blob/6659d42687fa92e892b605c3ac6ad856195d0b38/app/assessmentsList/get.controller.test.js#L50) to mock out methods on the services themselves. I quite like dependency injection but I wonder whether we could still use Jest mocking instead of needing to create subclasses, might be more "standard" looking for other JS devs.